src/options.js

```diff
@@ -1,133 +1,8 @@
-    description: "Makes the descendants tree on profile pages collapsible.",
-    category: "Profile",
-  },
-  {
-    name: "AKA Name Links",
-    id: "akaNameLinks",
-    description:
-      'Adds surname page links to the "aka" names on the profile page.',
-    category: "Profile",
-  },
-  {
-    name: "Family Timeline",
-    id: "familyTimeline",
-    description:
-      "Displays a family timeline. A button is added to the profile submenu.",
-    category: "Profile",
-  },
-  {
-    name: "Draft List",
-    id: "draftList",
-    description:
-      "Adds a button to the Find menu to show your uncommitted drafts.",
-    category: "Global",
-  },
-  {
-    name: "Random Profile",
-    id: "randomProfile",
-    description: "Adds a Random Profile link to the Find menu.",
-    category: "Global",
-  },
-  {
-    name: "Distance and Relationship",
-    id: "distanceAndRelationship",
-    description:
-      "Adds the distance (degrees) between you and the profile person and any relationship between you.",
-    category: "Profile",
-  },
-  {
-    name: "Locations Helper",
-    id: "locationsHelper",
-    description:
-      "Manipulates the suggested locations, highlighting likely correct locations," +
-      " based on family members' locations, and demoting likely wrong locations, based on the dates.",
-    category: "Editing",
-  },
-
-  {
-    name: "Dark Mode",
-    id: "darkMode",
-    description: "Make WikiTree dark.",
-    category: "Style",
-  },
-  {
-    name: "Family Group",
-    id: "familyGroup",
-    description:
-      "Display dates and locations of all family members. A button is added to the profile submenu.",
-    category: "Profile",
-  },
-  {
-    name: "Automatic GEDCOM Cleanup (AGC)",
-    id: "agc",
-    description:
-      "Reformats a biography and updates data fields when the profile was created from a GEDCOM.",
-    category: "Editing",
-  },
-  {
-    name: "BioCheck",
-    id: "bioCheck",
-    description: "Check biography style and sources.",
-    category: "Editing",
-  },
-  {
-    name: "Category Finder Pins",
-    id: "categoryFinderPins",
-    description:
-      "Adds pins to Category Finder results (on the edit page), similar to the pins in the location dropdown.  These pins link to the category page for you to check that you have the right category.",
-    category: "Editing",
-  },
-  {
-    name: "Sort Badges",
-    id: "sortBadges",
-    description: "Buttons to move or hide your Club 100/1000 badges.",
-    category: "Other",
-  },
-  {
-    name: "Verify ID",
-    id: "verifyID",
-    description:
-      "When attaching a person by ID, you can see some details of the person and check that you've entered the correct ID.",
-    category: "Editing",
-  },
-];
+import { features, OptionType } from "./core/options/options_registry.mjs";
+import "./features/register_feature_options.mjs";
+
 
 // Categories
 const categories = ["Global", "Profile", "Editing", "Style"];
```

Comment:
The "Other" category must have gotten removed accidentally

Reply:
I don't think it was ever there. If you look at the lines below that:
```
// If a new feature is added with a new category, add the category to the list
features.forEach(function (feature) {
  if (!categories.includes(feature.category)) {
    categories.push(feature.category);
  }
});
```
It has always handled categories that are not in the initial initialization of `categories`

Reply:
This commit added "Other":
https://github.com/wikitree/wikitree-browser-extension/commit/93dc7faf41488520cf4bd1fb5185ae217078d5bc
@shogenapps

Reply:
Oh, yes. It was between your changes and mine. It works fine as it is for now. We can decide if we want to move to a different system of treating the category like an enum. We may also want to separate the category id from the category text/description. (see my notes on https://github.com/wikitree/wikitree-browser-extension/issues/87)